### ISSUE TITLE
chifra scrape - updating the Unchained Index should be automated

### ISSUE THREAD

**Issue by tjayrush** [TrueBlocks/trueblocks-core#3476]
Once #3788 is ready, we will have the ability to send transactions. 

When the index is updated, we check if pinning is on and use `state --send` feature to update Unchained Index smart contract.

Questions:
1. When and how often do we want to update the smart contract? Updating after a single chunk is written might be wasting ETH on gas fees. Should we have a time based logic, e.g. update every day at 23:59?
2. Should we check not only if pinning is on, but if the new chunks and manifest file are pinned?

**Comment by dszlachta** [comment]
Closing in favor of #3787
